llvm/lib/Target/AMDGPU/SILowerSGPRSpills.cpp

```diff
@@ -442,12 +440,5 @@ bool SILowerSGPRSpills::runOnMachineFunction(MachineFunction &MF) {
   SaveBlocks.clear();
   RestoreBlocks.clear();
 
-  // Updated the reserved registers with any physical VGPRs added for SGPR
```

Comment:
The phys VGPRs used for CSR SGPR spilling shouldn't be reused during SIPreAllocateWWMRegs.
I hope they will be marked reserved while getting the AllocationOrder from `RegClassInfo.runOnMachineFunction(MF)` while computing the AllocationOrder.
@arsenm any specific reason they are marked Reserved here explicitly?